src/components/header.tsx

```diff
@@ -88,7 +90,7 @@ function Header(props: HeaderProps) {
           height="24"
         />
       ),
-      visible: config.experiment,
+      visible: true, //data?.user_type === UserType.Individual || data?.user_type === UserType.Team,
```

Comment:
keep `experiment` .
`experiment` indicates if it is shown in production.
It does mean, actually, it does not complete the page because it is not integrated with APIs, right?

It's the reason why it should keep `experiment` for visible.